.github/workflows/regression.yml

```diff
@@ -5,7 +5,7 @@ jobs:
   build:
     strategy:
       matrix:
-        os: [ubuntu-latest, macOS-latest, windows-latest]
```

Comment:
Can you make this a separate PR, please? Just to help keep track of the fact that Windows isn't tested in CI/CD.